components/script/dom/servoparser/mod.rs

```diff
@@ -100,16 +102,22 @@ enum LastChunkState {
 }
 
 impl ServoParser {
-    pub fn parse_html_document(document: &Document, input: DOMString, url: ServoUrl) {
-        let parser = ServoParser::new(document,
-                                      Tokenizer::Html(self::html::Tokenizer::new(document, url, None)),
+    pub fn parse_html_document(document: &Document, input: DOMString, url: ServoUrl, async: bool) {
```

Comment:
No boolean argument for that please, and I think there is no reason an HTML document should be parsed in a sync way.